### PR TITLE
fix influx query error #6056

### PR DESCRIPTION
**Pull request by zhe1926**
- [ ] Add a unit test to verify that the fix works.
- [X] Explain briefly why the bug exists and how to fix it.
image

SearchService method use influxdb identifier in sql when we use storage-influxdb-plugin.Put influxdb identifier in the middle of double quotation marks can solve this question.

### CODE REVIEW

**Comment by wu-sheng** [comment]
Could you update the `CHANGES.md` in the root folder.

File: oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/MetadataQuery.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public Service searchService(String serviceCode) throws IOException {
         final WhereQueryImpl<SelectQueryImpl> whereQuery = select(ID_COLUMN, NAME, ServiceTraffic.GROUP)
             .from(client.getDatabase(), ServiceTraffic.INDEX_NAME)
             .where(eq(TagName.NODE_TYPE, String.valueOf(NodeType.Normal.value())));
-        whereQuery.and(eq(ServiceTraffic.NAME, serviceCode));
+        whereQuery.and(eq("\"" + ServiceTraffic.NAME + "\"", serviceCode));
```
Comment (dmsolr) [inline oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/MetadataQuery.java:128]: We have a constant with quotes, `InfluxConstants.NAME`.

Comment (zhe1926) [reply oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/MetadataQuery.java]: thanks,i submit again.

**Review by wu-sheng — APPROVED** [review]
LGTM

**Review by dmsolr — APPROVED** [review]

**Pull request merged**